x-pack/plugins/apm/public/components/routing/templates/apm_service_template/index.tsx

```diff
@@ -213,6 +213,22 @@ function useTabs({ selectedTab }: { selectedTab: Tab['key'] }) {
     query: queryFromUrl,
   } = useApmParams(`/services/{serviceName}/${selectedTab}` as const);
 
+  const { data: serviceAlertsCount = { alertsCount: 0 } } = useFetcher(
+    (callApmApi) => {
+      return callApmApi(
+        'GET /internal/apm/services/{serviceName}/alerts_count',
+        {
+          params: {
+            path: {
+              serviceName,
+            },
+          },
+        }
+      );
+    },
```

Comment:
Doesn't this need a time range? Does the alerts table that we display accept a time range? If so, will the user run into a scenario where we display a number of active alerts, but when they click on the tab, they don't see any alerts?

Reply:
We are not sending a time range to our alerts table.

Reply:
Perhaps the api should be changed to `/active_alerts_count` instead of `/alerts_count`.
For active alerts counts I don't think time range plays a part. We should get the number of currently active alerts regardless when they started occurring.